python/cudf/cudf/_lib/utils.pyx

```diff
@@ -153,7 +154,18 @@ cpdef generate_pandas_metadata(Table table, index):
     md_dict = json.loads(metadata[b"pandas"])
 
     # correct metadata for list and struct types
```

Comment:
```suggestion
    # correct metadata for list and struct types and taking care of pandas Nullable types naming.
```

Reply:
I actually had changed this but lost it when I did some undo-redo.